README.md

```diff
@@ -52,6 +55,9 @@ The javascript code uses grunt as a build tool. See `Gruntfile.js` for build dep
 Tests are written in jest, also utilizing puppeteer for end-to-end tests.  Be
 sure to run `npm install && grunt` before running the tests with `npm test`.
 
+NOTE:  tests fail intermittently, both on local environments and in circleci.
```

Comment:
Should this comment be repeated here?

Reply:
@ndushay up to you if you need to nuke this pre-merge

Reply:
I left it in on purpose;  I wasn't sure if the duplication was annoying or useful (top of file and in section about tests)